crates/payload/builder/src/lib.rs

```diff
@@ -28,7 +28,7 @@
 //! use std::pin::Pin;
 //! use std::task::{Context, Poll};
 //! use reth_payload_builder::{EthBuiltPayload, KeepPayloadJobAlive, EthPayloadBuilderAttributes, PayloadJob, PayloadJobGenerator};
-//! use reth_payload_builder::error::PayloadBuilderError;
+//! use reth_payload_primitives::PayloadBuilderError;
```

Comment:
we should re-export the PayloadBuilderError in L125 then this wouldn't break too much and some crates don't require an additional dep